iphone/Classes/TiMediaVideoPlayerProxy.m

```diff
@@ -702,9 +702,13 @@ - (NSNumber *)repeatMode
 
 - (id)naturalSize
 {
+  CGSize size = CGSizeZero;
+  if (movie && [movie player] && [[movie player] currentItem]) {
```

Comment:
ObjC supports optional chaining, so `if (movie.player.currentItem)` should work fine, even if `movie.player` is `nil.`

Reply:
I was looking for Axway coding style conventions about "dots vs brackets" (didn't find any), but instead I found [this](https://docs.axway.com/bundle/Titanium_SDK_allOS_en/page/objective-c_and_objective-c___coding_standards.html):
>
> It is illegal to use a statement such as {{ if (objcObject) { ... } }}. Instead directly compare to nil, only where required.

So, is `if (movie.player.currentItem)` OK?

Reply:
Well, it's really up to the developer. Nowadays, the dot-notation is preferred (introduced in ObjC 2.0+), but it's not part of the linting / clang-format. In regards to the nil check, the `movie.player.currentItem` is more strict, like `===` on JS.

Reply:
OK. Thank you, `movie.player.currentItem` it is.